src/BinSkim.Sdk/CompilerDataLogger.cs

```diff
@@ -19,10 +19,11 @@ public class CompilerDataLogger
         private const string CompilerEventName = "CompilerInformation";
 
         private readonly bool appInsightsRegistered;
-
         private readonly string sha256;
         private readonly string relativeFilePath;
 
+        private static string s_sessionId;
```

Comment:
>s_sessionId [](http://example.com/codeflow?start=30&length=11)

s_sesionId is a guid that will be used to know what was the session that generated the results. The reason behind this is because an analysis can take time and we want to be able to distinguish one run from another.